static/app/actionCreators/modal.tsx

```diff
@@ -51,6 +51,14 @@ type emailVerificationModalOptions = {
   actionMessage?: string;
 };
 
+type inviteMembersModalOptions = {
+  onClose?: () => void;
+  initialData?: {
```

Comment:
for the type of initialData you can copy the type from the modal itself: `Partial<InviteRow>[];`:
https://github.com/getsentry/sentry/blob/1eab00b6ce2406ba31e0addbba0669f1e8b95dbb/static/app/components/modals/inviteMembersModal/index.tsx#L29

just have to import InviteRow
import {InviteRow} from 'app/components/modals/inviteMembersModal/types';